docs/how_to/vector_stores.md

```diff
@@ -0,0 +1,46 @@
+# Using Vector Stores
```

Comment:
for a later PR: i think this page would benefit from some diagrams to show the differences between how gpt index interacts with the vector stores.

made an for later https://github.com/jerryjliu/gpt_index/issues/109

Reply:
Yeah totally!